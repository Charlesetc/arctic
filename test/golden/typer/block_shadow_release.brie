[
  define x 3
  : x [ x a ]
  x
]
