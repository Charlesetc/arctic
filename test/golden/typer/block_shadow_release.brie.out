{[4, 3]=>Open_function, [0, 1, 2, 5, 8]=>Literal(subtype=integer), [6, 7, 10]=>Function_literal(takes=3, returns=9)}